src/libraries/Microsoft.Extensions.DependencyModel/src/Microsoft.Extensions.DependencyModel.csproj

```diff
@@ -27,15 +27,16 @@ Microsoft.Extensions.DependencyModel.DependencyContext</PackageDescription>
     <InternalsVisibleTo Include="Microsoft.Extensions.DependencyModel.Tests" /> 
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <Reference Include="System.Runtime" />
-    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="$(SystemRuntimeInteropServicesRuntimeInformationVersion)" />
```

Comment:
I believe this should stay hard-coded to `4.3.0`. We don't want it upgraded to a newer version.

Reply:
If we service it we do, right?  Imagine we find a critical bug in that version.  We'd want to update all folks in the repo that reference it to the serviced version.

Reply:
I looked again. At first I thought this was an OOB library that could get newer versions - `4.7.0`, `5.0.0`, etc. But I see that it has been "dead ended" for new functionality and we will only ship new servicing releases of it.

Given that we would only ever service this package, then I agree this is the right change.